util/README_mtt_update.md

```diff
@@ -0,0 +1,213 @@
+# `mtt_update.py`—Minetest Translation Updater
+
+This Python script is intended for use with localized Minetest mods, i.e., mods that use
+`*.tr` and contain translatable strings of the form `S("This string can be translated")`.
+It extracts the strings from the mod's source code and updates the localization files
+accordingly. It can also be used to update the `*.tr` files in Minetest's `builtin` component.
+
+## Preparing your source code
+
+This script makes assumptions about your source code. Before it is usable, you first have
+to prepare your source code accordingly.
+
+### Choosing the textdomain name
+
+It is recommended to set the textdomain name (for `minetest.get_translator`) to be identical
+of the mod name as the script will automatically detect it. If the textdomain name differs,
+you may have to manually change the `# textdomain:` line of newly generated files.
+
+**Note:** In each `*.tr` file, there **must** be only one textdomain. Multiple textdomains in
+the same file are not supported by this script and any additional textdomain line will be
+removed.
+
+### Defining the helper functions
+
+In any source code file with translatable strings, you have to manually define helper
+functions at the top with something like `local S = minetest.get_translator("<textdomain>")`.
+Optionally, you can also define additional helper functions `FS`, `NS` and `NFS` if needed.
+
+Here is the list of all recognized function names. All functions return a string.
+
+* `S`: Returns translation of input. See Minetest's `lua_api.txt`. You should always have at
```

Comment:
Wrong file extension:

```suggestion
* `S`: Returns translation of input. See Minetest's `lua_api.md`. You should always have at
```

Also, it might be useful to make this a link.